containers/sprm-to-anndata/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Using Conda because pyarrow did not install easily on python base images.
+FROM continuumio/miniconda3:4.7.12
+
+# For tiff packages
+RUN apt-get update &&\
+      apt-get install -y gcc python3-dev
```

Comment:
I think it's a common practice to include something like `&& rm -rf /var/lib/apt/lists/*` on your apt-get line, so the cached layer is smaller.